test/__snapshots__/index.test.js.snap

```diff
@@ -57,15 +67,15 @@ exports[`show can handle multiple specified targets 1`] = `
 
 exports[`show can handle multiple targets 1`] = `
 "
-            <div data-toggle-name=\\"test\\"></div>
+            <div data-toggle-name=\\"test\\" class=\\"\\"></div>
```

Comment:
We had to update these snapshots to remove the empty classes in order to fix the failing unit tests, not sure how they've made their way back in?

Reply:
Just checked Travis and the unit tests have failed on these lines.

Reply:
Yep I was just looking at these. Strange as I've not edited the code that would make this change. I'll revert the 2 instances.